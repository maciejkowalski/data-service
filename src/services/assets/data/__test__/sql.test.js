const sql = require('../sql');

test('Assets knex should build the sql query', () => {
  expect(sql(['q', 'w', 'e'])).toMatchSnapshot();
});
