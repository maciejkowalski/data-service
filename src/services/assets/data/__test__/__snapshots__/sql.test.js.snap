// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Assets knex should build the sql query 1`] = `"select * from \\"assets\\" where \\"asset_id\\" in ('q', 'w', 'e')"`;
